lib/perseo.js

```diff
@@ -98,6 +100,22 @@ function start(callbackStart) {
         versionRoutes.AddTo(app);
         metricsRoutes.AddTo(app);
 
+        const options = {
```

Comment:
Could you elaborate on the swagger-related changes, please?

Anyway, version should be a fixed value. It should be got from packages.json.